package.json

```diff
@@ -31,6 +31,9 @@
       "request": "onDemand",
       "description": "Extension may require your trust on workspace for code execution."
     },
+    "capabilities": {
```

Comment:
BTW could you update the indentation to 4 spaces? including the above `workspaceTrust`

Reply:
`workspaceTrust` part is out-of-date, I'll update it in a separate PR.